src/arc.ts

```diff
@@ -96,6 +96,14 @@ export class Arc {
       {
         daos {
           id
+          name
```

Comment:
why?

Reply:
I wanted to cache it so it will be available for the later calls.